src/lib/jsonParseV2/parseEntityObject.cpp

```diff
@@ -88,6 +89,10 @@ std::string parseEntityObject
     }
     else if (name == "idPattern")
     {
+      if (idPatternAllowed == true)
+      {
+        return ERROR_NOTIMPLEMENTED;
+      }
```

Comment:
I think it would be more clear in the inverse semantic for the bool variable. I men, this code to be:

```suggestion
      if (idPatternAllowed == false)
      {
        return ERROR_NOTIMPLEMENTED;
      }
```

changing in the other places to make it to work.

Reply:
Fixed in 425b48c